### ISSUE TITLE
Device shows as 'off' but actually 'on'

### ISSUE THREAD

**Issue by Yoiks** [Wizzard72/Domoticz-Unifi-Presence#23]
Hi, great plugin - thanks for making it available!

Quick question ...

Although the device status is 'on' in the Domoticz logs, it is showing as 'off' in the Domoticz panel - is that expected behaviour? It doesn't affect the script logic, but it would be nice to visually see that the device is present on the network.

![image](https://user-images.githubusercontent.com/50944216/135750889-763d867c-d5fe-4c1e-8fde-9104f35a015d.png)

![image](https://user-images.githubusercontent.com/50944216/135750928-3b6126c9-eaa4-4a16-bad2-63b6f4e0ea0b.png)

![image](https://user-images.githubusercontent.com/50944216/135751122-f0da74c0-6afd-432d-84e7-40c3f6c11b8f.png)

Running Domoticz and Unifi in docker containers on Ubuntu 20.04. 

Domoticz Version: 2021.1 (build 13628) 

Unifi 6.4.54 (jacobalberty/unifi docker image)

**Comment by 5bomen** [comment]
I have the same issue where I use the plug-in to detect the status of my TV. The distinctive situation here is that the TV is on a wired connection behind a Unifi Switch. Could this be the cause of the misrepresentation?

**Comment by Wizzard72** [comment]
The plugin doesn't detect and process wired devices.

**Comment by 5bomen** [comment]
Ah, that is a bummer. Can I put in a feature request? Or is this very hard to code?

**Comment by 5bomen** [comment]
> 



> Ah, that is a bummer. Can I put in a feature request? Or is this very hard to code?

I guess it is as simple as removing the check on "is_wired" in the request_online_phones function?

**Comment by Wizzard72** [comment]
If you don't use the plugin for the detection of phones, then you can remove the is_wired. But if you use it also for phones than it can take a long time before a phone device is turned off. This because the output of the API turns the is_wired to true. This was a few unifi versions ago. So maybe they have solved it.

**Comment by Wizzard72** [comment]
What is your use case for detecting the TV?

**Comment by 5bomen** [comment]
> 



> What is your use case for detecting the TV?

My amplifier sometimes doesn't switch off when the TV is switched off (even though it should). So then the amplifier stays on all night comsuming a lof of energy. I want to trigger a script for switching it off when the TV is off and the master selector of the amplifier is set to TV.

**Comment by Wizzard72** [comment]
Aha, ilI do the same, but i do it with a powerstrip meter, so if it's under a certain Watts then it power off.

**Comment by 5bomen** [comment]
> Aha, ilI do the same, but i do it with a powerstrip meter, so if it's under a certain Watts then it power off.

Yes, I recognize the solution. But it is not smart enough for my situation. I could just be listening to the radio or Spotify over my sound system - in that case the amplifier should not be switched off.  The amplifier should only be switched of if the input selector is on TV input and the TV is off, not if any of the other inputs are selected, or if the TV is switched on.

**Comment by Wizzard72** [comment]
If my amp is in standby it uses about 30 Watts. If its on about 50 Watts. So that is easy. There are for Denon/Marantz and I think Onkyo plugins for domoticz. My Onkyo is too old :-( But I'm in the market for a new one. Do you have such a plugin running on your domoticz?

**Comment by 5bomen** [comment]
Yes I own a Onkyo TX-NR686. I use the Onkyo AV receiver (LAN) plugin.
